src/app/components/Dropdown.tsx

```diff
@@ -185,21 +185,22 @@ interface DropdownItemProps {
 }
 
 // tslint:disable-next-line:variable-name
-export const DropdownItem = ({message, href, prefix, onClick}: DropdownItemProps) => <li>
-  <FormattedMessage id={message}>
-    {(msg: Element | string) => href
-      ? <a href={href} onClick={onClick}>{prefix}{msg}</a>
-      /*
-        this should be a button but Safari and firefox don't support focusing buttons
-        which breaks the tab transparent dropdown
-        https://bugs.webkit.org/show_bug.cgi?id=22261
-        https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#Clicking_and_focus
-      */
-      // eslint-disable-next-line jsx-a11y/anchor-is-valid
-      : <a tabIndex={0} href='' onClick={onClick ? flow(preventDefault, onClick) : preventDefault}>{prefix}{msg}</a>
+export const DropdownItem = ({message, href, prefix, onClick}: DropdownItemProps) => <FormattedMessage id={message}>
+    {(msg: Element | string) => <li aria-label={assertString(msg, 'aria-label must be a string')}>
```

Comment:
You can just remove `Element` from type declaration if this should be always a string instead of `assertString`
However, I think a message for `aria-label` should have a separate prop which probably should be also optional because it is a component which is used in a lot of places